src/Form/Control/Password.php

```diff
@@ -4,7 +4,44 @@
 
 namespace Atk4\Ui\Form\Control;
 
+use Atk4\Ui\Button;
+use Atk4\Ui\Js\JsExpression;
+
 class Password extends Line
 {
     public string $inputType = 'password';
+
+    /**
+     * @var bool by setting this variable to true you get an Eye-button on the right side which can toggle the visibility of the entered password
+     */
+    public $eye;
+
+    protected function init(): void
```

Comment:
just do what the PHPStan tells you here :) (see other init() overrides)

for the second PHPStan error - I think you do not need a variable and do not want to what the JsExpression in an array

Reply:
Fixed...